cmd/pipeline/deploy.go

```diff
@@ -285,7 +286,8 @@ func waitForResourcesToBeRunning(ctx context.Context, name string, timeout time.
 				return err
 			}
 			allRunning := true
-			for _, status := range resourceStatus {
+			for resourceID, status := range resourceStatus {
```

Comment:
What is the value of resource ID?

Reply:
Can we use type/name instead? The resourceID is not human readable, and we don't expose it anywhere in the UI.